wallets/react-wallet-v2/.env.local.example

```diff
@@ -1,2 +1,7 @@
 NEXT_PUBLIC_PROJECT_ID=
 NEXT_PUBLIC_RELAY_URL=wss://relay.walletconnect.com
+NEXT_PUBLIC_HEDERA_ACCOUNT_ID=
+# WARNING: Do not use this approach of storing/accessing private keys in production.
+# This app is not configured with a secure storage mechanism, so this is just the
+# best way to make it work and showcase Hedera integration with WalletConnect.
+NEXT_PUBLIC_HEDERA_PRIVATE_KEY=
```

Comment:
```suggestion
# !!! WARNING!!!: 
# Do not use this approach of storing/accessing private keys in production.
# This is for development purposes only, this key will be publicly available in an untrusted environment - the browser
# As this demo app is not configured with a secure storage mechanism, this method of adding a Hedera Testnet account is for demonstration purposes only!
# To create a Hedera Testnet account, proceed to https://portal.hedera.com
NEXT_PUBLIC_HEDERA_PRIVATE_KEY=
```

Reply:
added exclamation points to really call it out. Also, put where they can create a testnet account in another spot, and after reading fully...it's well called out in the README.md ..though still think it's good here